### ISSUE TITLE
Un seul contributeur ?

### ISSUE THREAD

**Issue by stenier-nws** [DeniceDaCosta/Charte#1]
Bonjour,

Les 6 commit du projet proviennent du même contributeur

il est important que chacun contribue !

des commit réguliers vous permettront certainement d'arriver rapidement à un premier "conflit" Git - idéalement demain pendant que je serai parmi vous :)

**Comment by DeniceDaCosta** [comment]
Oui, on a eu des problèmes liée aux commit sur chacun de nous et du coup on a du tout faire sur un seul code ensemble.

Da Costa Dénice
________________________________
De : stenier-nws <notifications@github.com>
Envoyé : Wednesday, October 16, 2019 7:30:31 PM
À : DeniceDaCosta/Charte <Charte@noreply.github.com>
Cc : Subscribed <subscribed@noreply.github.com>
Objet : [DeniceDaCosta/Charte] Un seul contributeur ? (#1)


Bonjour,

Les 6 commit du projet proviennent du même contributeur

il est important que chacun contribue !

des commit réguliers vous permettront certainement d'arriver rapidement à un premier "conflit" Git - idéalement demain pendant que je serai parmi vous :)
